rosdep/base.yaml

```diff
@@ -4670,6 +4670,9 @@ mosquitto-dev:
   debian: [libmosquitto-dev]
   fedora: [mosquitto-devel]
   ubuntu: [libmosquitto-dev]
+mosquittopp-dev:
+  debian: [libmosquittopp-dev]
+  ubuntu: [libmosquittopp-dev]
```

Comment:
We've agreed that this should be a separate key like this.  We also know that Fedora bundles mosquittopp inside of mosquitto-devel.

With those two pieces of information, I think we should make the Fedora key for `mosquittopp-dev` the following:

```
  fedora: [mosquitto-devel]
```

That way it will still resolve (even though it would be a duplicate of mosquitto-dev).

Reply:
I've gone ahead and done this now.